### DISCUSSION TITLE
Guidance for porting Fomily to run on NodeJS/关于在NodeJS上运行formily的建议

### DISCUSSION THREAD

**Discussion by zhenchuan9r** [alibaba/formily#4127] [Q&A 2.x]
We are running Formily-designed forms on the frontend.
However, as the validation logics of the form is heavy, we don't want to implement them again on the backend.
Thus we are considering run formily on the backend-side NodeJS.
Thanks if we can receive some suggestions from the community and the development team.

我们使用Formily在前端运行着表单，但是相关校验如果在后端重新实现，具有较高的开发成本。
因此，打算在后端NodeJS上跑Formily代码，希望社区和官方能提供一些建议。

**Comment by franckXu** [comment]
用jsdom?
